leptos_dom/src/node_ref.rs

```diff
@@ -65,9 +75,13 @@ pub struct NodeRef<T: ElementDescriptor + 'static>(
 ///     }
 /// }
 /// ```
+#[track_caller]
 #[inline(always)]
 pub fn create_node_ref<T: ElementDescriptor + 'static>() -> NodeRef<T> {
-    NodeRef(create_rw_signal(None))
+    NodeRef {
+        defined_at: std::panic::Location::caller(),
```

Comment:
Needs `#[cfg(debug_assertions)]` on the `defined_at` field, otherwise this won't compile in release I don't think.

Reply:
Thanks... I will fix that in the next PR.